.circleci/config.yml

```diff
@@ -9,6 +9,9 @@ jobs:
       - run: npm install
       - run: npm run build
       - run: npm run test
+      - run:
+          name: Installing ts-node
+          command: npm install --no-save ts-node
```

Comment:
The problem reported by me is not just a CI issue, but also a local thing that happens when I try to use the app-toolbelt locally in the entitlementcard project, so this is not a sufficient fix. Furthermore, adding `ts-node` to the CI won't fix it in the CI for the entitlementcard :/ I will have a look whether perhaps https://github.com/digitalfabrik/entitlementcard/pull/2023 is already enough to fix the problem.

Reply:
Because we will eventually migrate to esm at some point even there is a ticket at integreat about migrating to esm.

Reply:
Fair point :+1: